homeassistant/components/pi_hole/binary_sensor.py

```diff
@@ -123,10 +124,9 @@ def __init__(
         self.entity_description = description
 
         if description.key == "status":
+            self._attr_has_entity_name = False
```

Comment:
The name translation won't get activated unless `has_entity_name` is `True`. What's happening here?